src/content/docs/opentelemetry/best-practices/opentelemetry-best-practices-metrics.mdx

```diff
@@ -449,9 +449,16 @@ Buckets with boundaries at negative and positive infinity are represented in New
 
 ### OTLP summary metrics [#otlp-summary]
 
-OTLP [summary metrics](https://github.com/open-telemetry/opentelemetry-proto/blob/a05597bff803d3d9405fcdd1e1fb1f42bed4eb7a/opentelemetry/proto/metrics/v1/metrics.proto#L254-L256) are similar to histograms in that they summarize a population of measurements, including sum and count. However, where histograms include buckets to describe the distribution of measurements, summaries include quantiles. These quantiles are of limited use since they cannot undergo spatial or temporal reaggregation. Summaries included in OpenTelemetry for legacy support, and OpenTelemetry APIs and SDK do not produce summaries.
+OTLP [summary metrics](https://github.com/open-telemetry/opentelemetry-proto/blob/a76fe9dea26871e8a6c494024bc9927fe73b8142/opentelemetry/proto/metrics/v1/metrics.proto#L274-L276) are similar to histograms in that they summarize a population of measurements, including sum and count. However, where histograms include buckets to describe the distribution of measurements, summaries include quantiles. These quantiles are of limited use since they cannot undergo spatial or temporal reaggregation. Summaries included in OpenTelemetry for legacy support, and OpenTelemetry APIs and SDK do not produce summaries.
 
-Summaries are translated to New Relic [`summary`](/docs/data-apis/understand-data/metric-data/metric-data-type/#metric-types).
+Summaries are translated to New Relic [`summary`](/docs/data-apis/understand-data/metric-data/metric-data-type/#metric-types). Note that the New Relic summary type does not support quantiles.
+
+<Callout variant="important">
+While summaries are ingested and translated to a New Relic [`summary`](/docs/data-apis/understand-data/metric-data/metric-data-type/#metric-types), they are not properly supported.
+New Relic assumes summaries represent a delta since the last measurement when in fact they are cumulative metrics (see: [aggregation temporality](#aggregation-temporality).
```

Comment:
```suggestion
New Relic assumes summaries represent a delta since the last measurement when in fact they are cumulative metrics (see [aggregation temporality](#aggregation-temporality).
```